### PR TITLE
Add validation & file upload

### PR DESCRIPTION
**Pull request by vitaliemiron**

### CODE REVIEW

File: apps/dashboard/forms.py [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
         fields = ["title", "thumbnail", "content", "url"]
+
+    def clean_title(self):
+        title = self.cleaned_data.get("title", False)
+        if len(title.split()) < 2:
+            raise forms.ValidationError("Title should contain at least 2 words")
+        return title
+
+    def clean_url(self):
+        url = self.cleaned_data.get("url", False)
```
Comment (poncho7rm) [inline apps/dashboard/forms.py]: Verifica ce arunca URLValidator cand ii dai valori gen None, bool sau int

Comment (vitaliemiron) [reply apps/dashboard/forms.py]: ,,Введите корректный URL"

File: apps/dashboard/forms.py [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+        validate_url = URLValidator()
+        validate_url(url)
```
Comment (poncho7rm) [inline apps/dashboard/forms.py]: Poti sa scrii asta ca URLValidator()(url) ca sa nu creezi instanta

File: apps/dashboard/views.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ class DashHome(LoginRequiredMixin, View):
 
     def get(self, request):
         user = request.user
-        book = Bookmark.objects.all().filter(author=request.user)
+        book = Bookmark.objects.all().filter(author=request.user).order_by("-pub_date")
```
Comment (poncho7rm) [inline apps/dashboard/views.py]: Cand chemi, .filter(), .order(), etc poti sa omiti .all()
Bookmark.objects.filter(author=request.user).order_by("-pub_date")

File: apps/dashboard/forms.py [hunk 4]
```diff
@@ -1,8 +1,21 @@
...
+from django.core.validators import URLValidator, RegexValidator
+from django.core.exceptions import ValidationError
 
 
 class BookmarkForm(forms.ModelForm):
     class Meta:
         model = Bookmark
         fields = ["title", "thumbnail", "content", "url"]
+
+    def clean_title(self):
```
Comment (aspatari) [inline apps/dashboard/forms.py]: use inline validators

**Pull request merged**